Doc/howto/logging-cookbook.rst

```diff
@@ -332,10 +332,10 @@ Suppose you configure logging with the following JSON:
         }
     }
 
-This configuration does *almost* what we want, except that ``sys.stdout`` would
-show messages of severity ``ERROR`` and above as well as ``INFO`` and
-``WARNING`` messages. To prevent this, we can set up a filter which excludes
-those messages and add it to the relevant handler. This can be configured by
+This configuration does *almost* what we want, except that ``sys.stdout`` would show messages
+of severity ``ERROR`` and only events of this severity and higher will be tracked
+as well as ``INFO`` and ``WARNING`` messages. To prevent this, we can set up a filter which 
```

Comment:
kindly reminder here :)

PR authors need to at least make sure all the CI checks are passed before asking for reviews. You can find the CI status at the bottom of the PR page, and clicking on the `details` of the failed jobs will lead you to the logs.

<img width="907" alt="image" src="https://github.com/python/cpython/assets/24987826/2c1a254b-14f5-43b3-8027-3913115cc7cc">
<img width="825" alt="image" src="https://github.com/python/cpython/assets/24987826/a9f68d9f-1209-44b6-8d7d-2c63d1f05baa">

The logs said that there is a trailing whitespace that needs to be removed. The following change should fix it.

```suggestion
as well as ``INFO`` and ``WARNING`` messages. To prevent this, we can set up a filter which
```